TA-windows-firewall-status-check/README.md

```diff
@@ -0,0 +1,92 @@
+# Windows Firewall Status Check Add-on for Splunk
+
+### Download from Splunkbase
+TODO
+
+
+OVERVIEW
+--------
+The Windows Firewall Status Check Add-on for Splunk allows users to check their Windows machine's firewall status. It uses the PowerShell script to collect the latest status of Windows Firewall based on Netsh command. The Add-on does not contain any dashboards or savedsearches.
+
+Install the Cyences App for Splunk (https://splunkbase.splunk.com/app/5351/) to get alerts and dashboards related to the data collected by this Add-on.
+
+
+* Author - CrossRealms International Inc.
+* Creates Index - False
+* Compatible with:
+   * Splunk Enterprise version: 9.0.x, 8.2.x
+   * OS: Platform Independent
+   * Browser: Does not have UI.
+
+
+
+TOPOLOGY AND SETTING UP SPLUNK ENVIRONMENT
+------------------------------------------
+There are two ways to setup this app:
+  1. Standalone Mode: 
+     * Install the `Windows Firewall Status Check Add-on`.
+  2. Distributed Mode:
+     * The Add-on is required on search head for field extraction.
+     * Install the `Windows Firewall Status Check Add-on` on the Universal Forwarders on Windows and configure it. (You could do that from Deployment Server.)
+     * Install the Add-on on a heavy forwarder if Universal Forwarder is forwarding to Heavy Forwarder. Otherwise, install it on an Indexers. Input configuration is not required for both Indexers and Heavy Forwarders.
+
+
+DEPENDENCIES
+------------------------------------------------------------
+* There are no external dependencies for this Add-on.
+
+
+INSTALLATION
+------------------------------------------------------------
+* From the Splunk Home page, click the gear icon next to Apps.
+* Click `Browse more apps`.
+* Search for `Windows Firewall Status Check Add-on`.
+* Click `Install`.
+* If prompted, restart Splunk.
+
+
+DATA COLLECTION & CONFIGURATION
+------------------------------------------------------------
+### Enable Data Inputs ###
+* Add the following stanzas in `TA-windows-firewall-status-check/local/inputs.conf` file and deploy it for all required Windows hosts.
+```
+[powershell://windows_firewall_status_check]
```

Comment:
stanza name needs to be updated to "powershell://windows_defender_firewall_status_check"

Reply:
I would change everything to this. (File name , input stanza and everything, don't include defender anywhere)
**windows_firewall_status_check**